h/services/search_index/service.py

```diff
@@ -2,7 +2,7 @@
 
 from h import storage
 from h.presenters import AnnotationSearchIndexPresenter
-from h.tasks.indexer import add_annotation, delete_annotation
+from h.tasks import indexer
```

Comment:
Importing `add_annotation`, `delete_annotation` and (now) `add_annotations_between_times` as top-level names was confusing because `service.py` also contains its own methods named `SearchIndexService.add_annotation()`, `add_annotation_by_id()`, `delete_annotation` and `add_annotations_between_times`. I think the code is clearer if, whenever it's accessing the `tasks.indexer` stuff, it's always `indexer.foo()`. This also enables the tests to have a patch fixture for this import and just name the patch fixture `indexer` (our usual naming style for patch fixture). You couldn't have patch fixtures named `add_annotation` and `delete_annotation`, or at least it'd be really confusing to, because one test class in `service_test.py` already contains fixtures with those names (that patch `SearchIndexService.add_annotation` and `delete_annotation`).

Reply:
Yes, we name it after the thing we're patching (so `indexer` in this case), that's what I meant by the style. The original motivation for this was simply to prevent proliferation of all kinds of invented fixture-naming styles by shortcircuiting the whole thing. Otherwise developers go nuts and will do `_indexer`, `indexer_`, `indexer_patch`, `mock_indexer`, `fake_indexer`, `stub_indexer`, `idx`, etc etc etc all in one file. Secondarily having the fixture name contain the name of the thing it patches makes it easier to guess what the fixture is without having to go and look